scripts/gitlab-mirror/mirror

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+TARGET=$1
+MIRROR=$2
+DELAY=${3:-60}
+SSH_KEY=${4:-/root/.ssh/gitlab-mirror}
+
+# Set up ssh and add a valid key
+eval `ssh-agent -s`
+ssh-add $SSH_KEY
+
+while true
+do
+mkdir -p repo/
+git clone --mirror $TARGET repo/
+cd repo
+git push --mirror $MIRROR
+cd ..
+rm -rf repo
```

Comment:
Perhaps we can use `git remote update` rather than removing the folder each time? (as per: https://stackoverflow.com/questions/6150188/how-to-update-a-git-clone-mirror )

Reply:
Good find, will try it out